python/Ganga/new_tests/GPI/newXMLTest/TestSJXMLGenAndLoad.py

```diff
@@ -241,6 +241,9 @@ def test_h_testXMLIndex(self):
             index_cat = raw_j._category
             this_index_cache = (index_cat, index_cls, index_cache)
 
+            print("just-built index: %s" % str(this_index_cache))
+            print("from disk: %s" % str(obj))
```

Comment:
It is defined as part of the Python language that `"%s" % a` is equivalent to `str(a)`. I would be *very* interested in any example where this does not work.